### DISCUSSION TITLE
Mute timings with Cron expressions (or bi-weekly)

### DISCUSSION THREAD

**Discussion by jeroenhabets** [grafana/grafana#60338] [Alerting]
Though "mute timings" is a great feature our bi-weekly releases cannot be configured

Now: Days of the week, Days of the month, Months, Years

Desired: either [cron expressions](https://en.wikipedia.org/wiki/Cron#Cron_expression) [Wikipedia] (known from crontab) or some other flexible way to specify the time interval supporting our bi-weekly use case.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
